test-framework/maven/src/main/java/io/quarkus/maven/it/MojoTestBase.java

```diff
@@ -268,7 +268,7 @@ public static void assertThatOutputWorksCorrectly(String logs) {
         assertThat(logs.contains(infoLogLevel)).isTrue();
         Predicate<String> datePattern = Pattern.compile("\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2},\\d{3}\\s").asPredicate();
         assertThat(datePattern.test(logs)).isTrue();
-        assertThat(logs.contains("features: [cdi, resteasy, undertow-websockets, vertx, vertx-web]")).isTrue();
+        assertThat(logs.contains("features: [cdi, resteasy, undertow-websockets]")).isTrue();
```

Comment:
They are now only displayed if the user-facing extensions are used. Otherwise it's just internal plumbing and the user shouldn't be aware of it.

Reply:
agreed.